dist/index.d.ts

```diff
@@ -0,0 +1 @@
+export { initialiseLoggingClient } from "./loggingClient";
```

Comment:
`dist` directory should be gitignored as it's build output

Reply:
doh